tests/Rule/RequirePreviousExceptionPassRuleTest.php

```diff
@@ -13,29 +13,29 @@
 class RequirePreviousExceptionPassRuleTest extends RuleTestCase
 {
 
-    private ?bool $checkRethrownExceptionAcceptsCaughtOne = null;
+    private ?bool $ignoreIfExceptionIsNotAcceptableByRethrownOne = null;
 
     protected function getRule(): Rule
     {
-        if ($this->checkRethrownExceptionAcceptsCaughtOne === null) {
+        if ($this->ignoreIfExceptionIsNotAcceptableByRethrownOne === null) {
             throw new LogicException('Testcase need to initialize this');
         }
 
         return new RequirePreviousExceptionPassRule(
             self::getContainer()->getByType(Standard::class),
-            $this->checkRethrownExceptionAcceptsCaughtOne,
+            $this->ignoreIfExceptionIsNotAcceptableByRethrownOne,
         );
     }
 
     public function testWithAcceptCheck(): void
     {
-        $this->checkRethrownExceptionAcceptsCaughtOne = true;
+        $this->ignoreIfExceptionIsNotAcceptableByRethrownOne = true;
         $this->analyseFile(__DIR__ . '/data/RequirePreviousExceptionPassRule/with-accept-check.php');
     }
 
     public function testWithoutAcceptCheck(): void
     {
-        $this->checkRethrownExceptionAcceptsCaughtOne = false;
+        $this->ignoreIfExceptionIsNotAcceptableByRethrownOne = false;
```

Comment:
I think the meaning is now opposite. You should set it to TRUE to test it without the check. Fix it everywhere pls.

Reply:
Renamed once to make it even less ambiguous.